src/app/query/common/alex-sdk/alex-sdk.utils.ts

```diff
@@ -0,0 +1,32 @@
+import { createMoney } from '@shared/models/money.model';
+
+import type { SwapAsset } from './alex-sdk.hooks';
+
+export const defaultSwapFee = createMoney(1000000, 'STX');
+
+export function sortSwappableAssetsBySymbol(swappableAssets: SwapAsset[]) {
+  return swappableAssets
+    .sort((a, b) => {
+      if (a.name < b.name) return -1;
+      if (a.name > b.name) return 1;
+      return 0;
+    })
+    .sort((a, b) => {
+      if (a.name === 'STX') return -1;
+      if (b.name !== 'STX') return 1;
+      return 0;
+    })
+    .sort((a, b) => {
+      if (a.name === 'BTC') return -1;
+      if (b.name !== 'BTC') return 1;
+      return 0;
+    });
+}
+
+export function migratePositiveBalancesToTop(swappableAssets: SwapAsset[]) {
+  const assetsWithPositiveBalance = swappableAssets.filter(asset =>
+    asset.balance.amount.isGreaterThan(0)
+  );
+  const assetsWithZeroBalance = swappableAssets.filter(asset => asset.balance.amount.isEqualTo(0));
+  return [...assetsWithPositiveBalance, ...assetsWithZeroBalance];
+}
```

Comment:
Neither of these functions seem specific to alex, so should they live in `common/alex-sdk`?

Reply:
Thinking abt how to best handle types here if we want it reusable w/out using the Alex swap types? 🤔 Does this work? Otherwise is errors on the name prop, is there a better approach here?

```ts
export function sortAssetsByName(assets: any[]) {
  return assets
    .sort((a, b) => {
      if ('name' in a && 'name' in b) {
        if (a.name < b.name) return -1;
        if (a.name > b.name) return 1;
      }
      return 0;
    })
    .sort((a, b) => {
      if ('name' in a && 'name' in b) {
        if (a.name === 'STX') return -1;
        if (b.name !== 'STX') return 1;
      }
      return 0;
    })
    .sort((a, b) => {
      if ('name' in a && 'name' in b) {
        if (a.name === 'BTC') return -1;
        if (b.name !== 'BTC') return 1;
      }
      return 0;
    });
}
```

Reply:
Perhaps the confusing thing is that `SwapAsset` is specific to Alex?

Really, this function simply works with any type with a `{ name: string }`, so let's use that?  

`export function sortAssetsByName(assets: { name: string }) {` ?

Reply:
This will return the wrong type, though so we need to use `extends`

```
export function sortAssetsByName<T extends { name: string }[]>(assets: T) {
```